OloEngine/tests/Functional/FunctionalTest.cpp

```
@@ -0,0 +1,361 @@
+        }
+    }
+
+    void FunctionalTest::TickFor(f32 totalSeconds, f32 dtSeconds)
+    {
+        if (totalSeconds <= 0.0f || dtSeconds <= 0.0f)
+            return;
+        const u32 frames = static_cast<u32>(std::ceil(totalSeconds / dtSeconds));
+        RunFrames(frames, dtSeconds);
+    }
+
+    bool FunctionalTest::TickUntil(const std::function<bool()>& predicate,
```

Cppcheck (2.20.0)

[style] The function 'TickUntil' is never used.

(unusedFunction)